Example/Playground.playground/Pages/Introduction.xcplaygroundpage/Contents.swift

```diff
@@ -1,14 +1,79 @@
+ 
+ ## What sets Genything apart?
+ The existing libraries for data data generation are usually specific to property based testing and have hard requirements. One limitation seen in current solutions is not being able to use the same generator in an example app and within unit tests. There is also no abstraction of randomness and limited support for result reproducibility.
+ - - -
+ */
+/*: ### What is a Generator?
+ A way to generate random values of a given type with a context that lets us decide how random we would like to be. */
 
-// Textual
+let boolGenerator = AnyGenerator<Bool> { context in /// `context` is a `RandomSource` which
+    Bool.random(using: &context.rng)                /// lets us control the randomness
+}
+/*: ### How can we create a generator?
+ The most common way to create a generator is using `arbitrary`. This property is available for most Swift types. We can also implement the `Arbitrary` protocol to unlock this property for our custom types. */
 
-// - What sets Genything apart?
-// - Motivation?
-// - What is it for?
+let digitGenerator: AnyGenerator<Int> = (0...9).arbitrary
 
-// Code
+/// If we want pseudo-realistic fake data we can import Trickery and use Fake.
+/// We can also use operators to compose complex generators. This will be looked into further in the next page of the playground.
+let fakeDigitGenerator: AnyGenerator<Int> = Fake.Numerics.digits
+/*: ### How can we get values out of a generator?
+ The main functions to extract values from a generator are:
+ * `next(_:)` which produces the next element from this generator
+ * `take(_:,randomSource:)` which returns an array of values from the generator
+ * `sequence(_:,randomSource:)` which returns a sequence of this generator's values which will produce `nil` after `size` iterations */
+/// A RandomSource is needed to provide a context with which to generate the random values.
+let randomSource = RandomSource.predetermined()
 
-// - What is a Generator?
-// - How can I create a generator?
-// - How can I get values out of a generator?
-// - What is a Random Source? How am I supposed to use it?
+let generatedInt = digitGenerator.next(randomSource) /// Returns the next value from the generator, here an integer
+
+let generatedArray = digitGenerator.take(5, randomSource: .predetermined()) /// Returns an array of 5 integers taken from the generator
+let generatedFiniteSequence = digitGenerator.sequence(5, randomSource: .predetermined()) /// returns a sequence of 5 integers.
```

Comment:
I'd consider making these two random sources the previously initialized `randomSource`. I think in general (considering uses beyond unit tests) it's more likely for people to require a shared source and to help them build that habit.

Reply:
ah ok, I see